myenumerable.rb

```diff
@@ -0,0 +1,31 @@
+module MyEnumerable
+  def my_all?
+    return unless block_given?
+
+    result = false
+    @list.each do |n|
+      result = true if yield(n)
+    end
+    result
+  end
+
+  def my_any?
+    return unless block_given?
+
+    any = false
+    @list.each do |n|
+      any = true if yield(n)
+    end
+    any
+  end
+
+  def my_filter?
+    return unless block_given?
+
+    number_array = []
+    @list.each do |n|
+      number_array.push(n) if yield(n)
```

Comment:
- [ ] Kindly make sure you use the same method name in your `list` as the ones in the `module.`
`.all?`, `.filter?` and `.any` instead of `.my_all?`, `my_each` and `.my_filter`
![image](https://user-images.githubusercontent.com/97100091/212480495-2347b9e7-6eb6-4231-8f4a-7c74861bab00.png)